packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -136,11 +137,31 @@ class RefreshIndicator extends StatefulWidget {
   /// Typically a [ListView] or [CustomScrollView].
   final Widget child;
 
-  /// The distance from the child's top or bottom edge to where the refresh
-  /// indicator will settle. During the drag that exposes the refresh indicator,
-  /// its actual displacement may significantly exceed this value.
+  /// The distance from the child's top or bottom [bottom][edgeOffset] where
```

Comment:
Everything in '[]' need to be proper identifiers as they are turned into links in the generated docs.

The '[bottom]' reference needs to be removed. It is not a property of the indicator so it will create a broken link in the documentation.  Not sure why this didn't fail a test as we have checks for that in place.

Reply:
@darrenaustin done! :)